sourcecode/hub/tests/Browser/ContentTest.php

```diff
@@ -942,6 +927,8 @@ public function testEditsContentViaLti(): void
                                 ->press('Send'),
                         ),
                 )
+                // FIXME: There must be something that can be checked for success/completion of Send?
```

Comment:
Perhaps `waitForText()`?

Reply:
This should result in the message `If you can see this, the LTI launch was successful.` ?
That text does not display. Seems that an error occurs in Edlib resource kit, it complains that `about:blank` is missing host and fails with a 500

Reply:
Looking at this again, I don't believe there's an issue using `press()` here. This is submitting a real form, not a JS imitation of one, so it shouldn't proceed until the response for the POST request is received.

Rather, the issue is likely with Meilisearch taking an indeterminate amount of time to index content, and the content not necessarily being ready immediately after it was submitted for indexing.

`content_item_return_url` could also be set to an actual http(s) URL if that makes thing easier. What's being tested here is just the middle step of the saving process, where Edlib saves the content, before redirecting back.